src/wazuh_modules/wm_ciscat.c

```diff
@@ -94,10 +125,28 @@ void* wm_ciscat_main(wm_ciscat *ciscat) {
     // Define path where CIS-CAT is installed
 
     if (ciscat->ciscat_path) {
-        snprintf(cis_path, OS_MAXSTR - 1, "%s", ciscat->ciscat_path);
+        switch (wm_relative_path(ciscat->ciscat_path)) {
+            case 0:
+                // Full path
+                snprintf(cis_path, OS_MAXSTR - 1, "%s", ciscat->ciscat_path);
+                break;
+            case 1:
+                // Relative path
+            #ifdef WIN32
+                if (current)
```

Comment:
`current` will always be a valid pointer, but it may be empty.
Should be `if (*current) {}`

Missing `else` part.
If `current` is empty, `cis_path` may not be initialized and copy trash data.
I think we may use a relative path if we can't get the current directory:
```c
else {
    strncpy(cis_path, ciscat->ciscat_path, OS_MAXSTR - 1);
    cis_path[OS_MAXSTR - 1] = '\0;
}
```